lib/data/kinetic/backend.js

```diff
@@ -0,0 +1,155 @@
+import crypto from 'crypto';
+import kinetic from 'kineticlib';
+import stream from 'stream';
+import net from 'net';
+
+import { errors } from 'arsenal';
+
+import config from '../../Config';
+
+let sequence = 1;
+
+const kDrives = {
+    port: config.kinetic ? config.kinetic.port : 8123,
+    host: config.kinetic ? config.kinetic.host : 'localhost',
+};
+
+function propError(pdu) {
+    const statusCode = pdu.getStatusCode();
+    let err;
+
+    if (statusCode !== kinetic.errors.SUCCESS) {
+        if (statusCode === kinetic.errors.NOT_FOUND) {
```

Comment:
I would have wrote this as follow:

```js
        if (statusCode === kinetic.errors.NOT_FOUND) {
            return errors.ObjNotFound;
        }
        return kinetic.getErrorName(statusCode);
```

And so the `err` variable becomes useless.